### DISCUSSION TITLE
tutorials/voyage-part2/06-subgraph-planning

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#135] [course feedback]
# tutorials/voyage-part2/06-subgraph-planning

https://www.apollographql.com/tutorials/voyage-part2/06-subgraph-planning

**Comment by stephan-buckmaster** [comment]
Why do you designate Host as an Entity? It is self-contained and has no fields depending on other types.

**Reply by lizhennessy** [reply]
Hey there @stephan-buckmaster ! Great question - in this case, we designate `Host` as an entity because we plan to define its fields across _multiple_ subgraphs. I like to think of it as, will multiple subgraphs need to have a "concept" of what a `Host` is? 🤔 Clearly we would have something in the `accounts` subgraph to designate someone as host/guest, but we will also use the concept of a `Host` in our `reviews` subgraph (where a `Host` can write a review about a particular guest!). The nice thing is transforming a type into an entity is a pretty quick process, and we can always remove the `@key` directive if/when we opt not to use the type in other subgraphs. Nice to have when we need it though!

**Reply by stephan-buckmaster** [reply]
Thanks @lizhennessy , makes sense -- And thanks to the whiole team for the tutorials! Learned a lot!
